.buildkite/package.json

```diff
@@ -15,7 +15,8 @@
     "globby": "^11.1.0",
     "js-yaml": "^4.1.0",
     "minimatch": "^5.0.1",
-    "tslib": "*"
+    "tslib": "*",
+    "bump-cli": "^2.8.2"
```

Comment:
The dependencies here are installed on every step and we want to keep it slim if possible.  Would it be possible to install this only on the publish step?

Mind if we fix the `npm audit` issues too?

Reply:
I've move this to an `npm install...` command in the script file. The audit issues went away, but I'm not quite sure how to go about addressing them now as there were issues about `lodash.template`?